### DISCUSSION TITLE
markdown数学公式无法预览吗?

### DISCUSSION THREAD

**Discussion by dualer** [walinejs/waline#309] [Q&A] [answered]
RT

**Comment by Mister-Hope** [comment, marked as answer]
RT: 😉
![image](https://user-images.githubusercontent.com/33315834/121665072-bbf4dc80-cada-11eb-927d-b5c98650e1b7.png)
链接: https://waline.js.org/guide/client/syntax.html#%E5%8F%97%E9%99%90%E7%9A%84%E9%A2%84%E8%A7%88%E5%8A%9F%E8%83%BD

**Reply by dualer** [reply]
好吧, 谢谢回复
